qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
Is this change necessary ?
If the default was v1.2 and we changed it to v1.3 then the apis which are written by customers might fail.